### PR TITLE
Fpki trust stores

### PR DESCRIPTION
**Pull request by clstmbrly**
@lachellel This is the updated fpki_truststores.md section for the FPKI-Guides.

### CODE REVIEW

File: pages/fpki_truststores.md [hunk 1]
```diff
@@ -4,43 +4,50 @@ title: Trust Stores
...
 
-This list of trusted root certificates are contained within what is known as a **Trust Store** in either an application or operating system. 
+Instead, each application and the operating system (OS) on your computer keep a list of **trusted root certificates** in a **Trust Store** within the specific application or the OS. From their list of trusted root certificates, the application or the OS issues certificates to people (i.e., identities) and devices. When your computer is presented with an identity or device certificate from a PIV credential, website, e-mail, or other digital item, either an application or the OS will verify whether or not the certificate relates ("chains") to one of the trusted root certificates in the Trust Store. (**Note:** Not only do applications and OSs keep lists of trusted root certificates--Trust Stores themselves sometimes keep a list of trusted root certificates that comply with the Trust Store's requirements.)
```
Comment (lachellel) [inline pages/fpki_truststores.md]: remove this line - it's confusing: 

> From their list of trusted root certificates, the application or the OS issues certificates to people (i.e., identities) and devices.

Remove the ** Note: ** sentence too.  Confusing.

File: pages/fpki_truststores.md [hunk 2]
```diff
@@ -4,43 +4,50 @@ title: Trust Stores
...
 
->**Note:** Google Chrome uses the underlying trust library of the operating system on Windows or Apple OS X systems.  Linux-based systems distribute the Mozilla NSS Library which may be modified by each version of Linux. Google is working on developing its own program similar to Mozilla.
 
 ## How do I manage a Trust Store?
 
-TODO:  Add the info from the trust store guide from FPKI
+Managing a Trust Store must be done via the command line or the Graphical User Interface (GUI) software tools.  With the GUI tools, you can  view, add, and delete CA certificates. **It is vital to carefully manage the CA certificates in a Trust Store**. To ensure careful management of CA certificates, you should keep only *needed* CA certificates and delete *unwanted* CA certificates. A *needed*, but missing, CA certificate could result in a Denial of Service (DoS), and the presence of an *unwanted* CA certificate could result in an attacker successfully gaining authorized access.
```
Comment (lachellel) [inline pages/fpki_truststores.md]: I would comment out this section for now (min viable).  I don't think we have audience correct yet.

Comment (clstmbrly) [reply pages/fpki_truststores.md]: Would this comment work: "This section will be added in a future update to the FPKI Guides."  Or should I delete the entire section (including heading)?

Comment (clstmbrly) [reply pages/fpki_truststores.md]: Commented out title and section for now.

File: pages/fpki_truststores.md (hunk 1 above)
Comment (clstmbrly) [reply pages/fpki_truststores.md]: Items were removed.

**Comment by clstmbrly** [comment]
@lachellel @djpackham The fpki_truststores.md has been updated per your changes.  Do you have any additional changes?  Also, I made a few small fixes (command fixes, etc., and a few changes from Omar).  Chunde made one link change according to what he understands Federalist syntax to be.  He was not able to test it.  (He will look at Jordan's method and GitHub.io site tomorrow.)

**Comment by clstmbrly** [comment]
@lachellel @djpackham The link issue has been resolved for fpki_truststores.md, and all changes have been incorporated.  Do you have any further changes?

**Comment by djpackham** [comment]
@clstmbrly - for MVP we'll trim some of the content back. Thx.

**Comment by clstmbrly** [comment]
@djpackham  Ok, thank you.  Please let me know if you need any assistance with this.

**Pull request closed without merging**